tests/ui/attributes/rustc_confusables.rs

```diff
@@ -0,0 +1,26 @@
+#![feature(rustc_attrs)]
```

Comment:
Can you add a second test that checks whether the attribute also works cross-crate? You can add new file in `./auxiliary` and then use `extern crate` here (or rather, in a second test file).
Not all attributes are persisted in metadata across crates so having a test for it is useful (you did it correctly and it should work, if you add `@only_local` to the attribute declaration the test should fail).

Reply:
I moved the declaration of and attributes on `BTreeSet` to the auxillary test file (should be counted as another crate right?) and kept the uses to the primary test file.